### PR TITLE
feat: document UAA DB issue

### PR DESCRIPTION
**Pull request by bruce-ricard**

### CODE REVIEW

**Review by anita-flegg — APPROVED** [review]
LGTM -- I made a minor wording edit.

File: runtime-rn.html.md.erb [hunk 1]
```diff
@@ -1407,5 +1407,5 @@ failed to connect: unable to lookup a log consumer
 
 The UAA can cause and experience slowness in its database when
 attempting to hit its `/users` endpoint. The issue is especially
-noticeable when the UAA contains a large number of users, like in the
+noticeable when the UAA contains a large number of users; for example, in the
 milions.
```
Comment (Tallicia) [inline runtime-rn.html.md.erb:1411]: should be millions.

File: runtime-rn.html.md.erb [hunk 2]
```diff
@@ -1402,3 +1402,10 @@ then Loggregator Agent jobs deployed by these products may repeatedly log a fail
 ```
 failed to connect: unable to lookup a log consumer
 ```
+
+### UAA causes database overload
```
Comment (swalchemist) [inline runtime-rn.html.md.erb:1406]: Consider mentioning that it only happens when using MySQL for UAA.

Also, are you adding this for 6.0.1?

File: runtime-rn.html.md.erb [hunk 3]
```diff
@@ -1402,3 +1402,10 @@ then Loggregator Agent jobs deployed by these products may repeatedly log a fail
...
+
+The UAA can cause and experience slowness in its database when
+attempting to hit its `/users` endpoint. The issue is especially
```
Comment (swalchemist) [inline runtime-rn.html.md.erb:1409]: It the endpoint /users or /Users, or is it case-insensitive?

File: runtime-rn.html.md.erb (hunk 2 above)
Comment (bruce-ricard) [reply runtime-rn.html.md.erb:1406]: Comment about MySQL added. 

I'll copy this to 6.0.1 as soon as the paragraph is approved and ready to go!

**Review by anita-flegg — COMMENTED** [review]
Looks good

**Pull request merged**